### DISCUSSION TITLE
ROW_NUMBER() with gino

### DISCUSSION THREAD

**Discussion by makupi** [python-gino/gino#752] [Q&A]
I want to order my table, index the row numbers and then select certain values - something like this:

```sql
SELECT
	*
FROM
	(
		SELECT
			product_id,
			product_name,
			price,
			ROW_NUMBER () OVER (ORDER BY product_name)
		FROM
			products
	) x
WHERE
	ROW_NUMBER BETWEEN 6 AND 10;
```

Is that possible with gino?

**Comment by wwwjfy** [comment]
gino is using the same syntax engine from sqlalchemy, which has a richer source of Q&A we can find from different forums.
It should help to find answers using sqlalchemy + the actual question
